Automattic/jetpack
# [RNmobile] Add native supported VideoPress player

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/5509



## Proposed changes:

Adds a minimally functioning native VideoPress player.

The changes here do not account for design and loading performance. Those updates will follow while the native block is still behind a feature flag.


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the mobile editor
* Open a post that contains a VideoPress block.
* Verify that the block renders the video thumbnail with out cropping (try both portrait and landscape videos )
* Touch the block, note that the player does not open.
* Once the block is selected try touching the video.
* The video should play fullscreen.

* **Note**: ~There is an [open issue on Android](https://github.com/wordpress-mobile/gutenberg-mobile/issues/5565) regarding the `Sandbox` component. For now test plan will only succeed completely on iOS.~ The Sandbox issue has been resolved so testing is no longer limited to iOS. I have noticed some odd behavior when touching the video on Android but I think we'll have to fix those issues in Gutenberg and should not block these changes.

* **Note**:  The scope of the changes here are limited to existing VideoPress blocks. A block can be added via the mobile app but the post should be reloaded before verifying the test steps. Alternatively a VideoPress can be added to a post via the Web.
